dev/validate_catalogs.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 # find catalogs in the dev folder
-PATH = "./dev"
+PATH = "./scivision/catalog/data"
```

Comment:
I assume this will be changed with #73 ?

Reply:
That will be the plan yes